llvm/tools/llvm-objcopy/ObjcopyOpts.td

```diff
@@ -24,17 +24,20 @@ def O : JoinedOrSeparate<["-"], "O">,
         Alias<output_target>,
         HelpText<"Alias for --output-target">;
 
-defm new_symbol_visibility : Eq<"new-symbol-visibility", "Visibility of "
-                                "symbols generated for binary input or added"
-                                " with --add-symbol unless otherwise"
-                                " specified. The default value is 'default'">;
+defm new_symbol_visibility
```

Comment:
Please avoid reformatting the entire file in the middle of another change. If necessary, create an NFC patch to do the reformatting and base this PR on top of that.